genesyscloud/webdeployments_configuration/utils/resource_genesyscloud_webdeployments_configuration_journey.go

```diff
@@ -0,0 +1,230 @@
+package webdeployments_configuration_utils
+
+import (
+	"terraform-provider-genesyscloud/genesyscloud/util/lists"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/mypurecloud/platform-client-sdk-go/v119/platformclientv2"
+)
+
+func readSelectorEventTriggers(triggers []interface{}) *[]platformclientv2.Selectoreventtrigger {
```

Comment:
We should probably call this `buildSelectorEventTriggers` instead of `readSelectorEventTriggers` to consist with the other packages. Same for the functions below

Reply:
Updated util methods for naming consistency